democracy/views/hearing.py

```diff
@@ -53,12 +53,42 @@ class HearingFilterSet(django_filters.rest_framework.FilterSet):
         field_name="labels__id", lookup_expr="in", distinct=True, widget=django_filters.widgets.CSVWidget
     )
     following = django_filters.BooleanFilter(method="filter_following")
+    open = django_filters.BooleanFilter(method="filter_open")
+    created_by = django_filters.CharFilter(method="filter_created_by")
 
     def filter_following(self, queryset, name, value):
         if value and self.request.user.is_authenticated:
             return queryset.filter(followers=self.request.user)
         return queryset
 
+    def filter_open(self, queryset, name, value):
+        if value:
+            return (
+                queryset.filter(close_at__gt=datetime.datetime.now())
+                .filter(open_at__lt=datetime.datetime.now())
+                .filter(force_closed=False)
+            )
+        else:
+            return (
+                queryset.filter(close_at__lt=datetime.datetime.now())
+                | queryset.filter(open_at__gt=datetime.datetime.now())
+                | queryset.filter(force_closed=True)
+            ).distinct()
```

Comment:
Could replace this with excludes, I think:
```suggestion
            return (
                queryset.exclude(close_at__gt=datetime.datetime.now())
                .exclude(open_at__lt=datetime.datetime.now())
                .exclude(force_closed=False)
            )
```

Reply:
Or something else than the current form